inst/NEWS.Rd

```diff
@@ -2,6 +2,12 @@
 
 \title{NEWS for GENESIS}
 
+\section{Version 2.33.2}{
+  \itemize{
+    \item Set extremely small p-values calculated with pchisq to \code{Machine$double.xmin}.
```

Comment:
@amstilp I think we could be more clear here that the issue is being returned as 0, not that we just picked some threshold for some reason. Maybe: 

Set extremely small p-values returned as 0 by pchisq to \code{Machine$double.xmin}.

Reply:
I would put it like this: "Set extremely small p-values calculated with pchisq to \code{Machine$double.xmin}. This change prevents GENESIS from returning p-values equal to 0."

because it's not just pval=0; it's any p-values less than Machine$double.xmin. So you could have pchisq return 1e-324 but then the .pchisq_filter_extreme would change it to the Machine$double.xmin value.

Reply:
Makes sense and works for me. Everything else about this update looks good to me.